range-v3/build2file

```diff
@@ -0,0 +1,53 @@
+intf_libs = # Interface dependencies.
+impl_libs = # Implementation dependencies.
+
+./: lib{range-v3}: include/{hxx ixx txx}{**} $impl_libs $intf_libs
+
+# Include the generated version header into the distribution (so that we don't
+# pick up an installed one) and don't remove it when cleaning in src (so that
+# clean results in a state identical to distributed).
+#
+lib{range-v3}: include/range/v3/hxx{version}
+include/range/v3/
+{
+  # @cja struggling to see how to do this in a nice way, but anyway, what is reason for this? seems version.hpp is already set in upstream.
+  hxx{version}: $src_base/../../../in{version} $src_root/manifest
+   #$src_base/in{version} $src_root/manifest
+  {
+    dist = true
+    clean = ($src_root != $out_root)
+    in.symbol = '@'
+    RANGE_V3_MAJOR = $version.major
+    RANGE_V3_MINOR = $version.minor
+    RANGE_V3_PATCHLEVEL = $version.patch
+  }
+}
+
+
+# @cja note: removed -I for /include/std, not sure what reason was for it's previous inclusion?
```

Comment:
I don't remember so it might have been a failed copy-paste. If it works without it feel free to not keep it.

Reply:
Yeah CI passed for me with this removed. So I'll remove for now, we'll see if it shows up again.